src/pages/home/home.ts

```diff
@@ -874,16 +874,16 @@ export class HomePage {
     });
   }
 
-  private showInfoSheet(params): void {
+  private showInfoSheet(altCurrency): void {
```

Comment:
I'd rename the function's name to be more specifics for "altCurrency". It's a generic name.